### PR TITLE
fastfetch: update binary URL

### PR DESCRIPTION
**Pull request by CarterLi**
Update the file name pattern of fastfetch to match the latest version.

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

File: bucket/fastfetch.json [hunk 1]
```diff
@@ -21,10 +21,10 @@
     "autoupdate": {
         "architecture": {
             "64bit": {
-                "url": "https://github.com/fastfetch-cli/fastfetch/releases/download/$version/fastfetch-$version-Win64.7z"
+                "url": "https://github.com/fastfetch-cli/fastfetch/releases/download/$version/fastfetch-$version-windows-amd64.7z"
             },
             "32bit": {
-                "url": "https://github.com/fastfetch-cli/fastfetch/releases/download/$version/fastfetch-$version-Win32.7z"
+                "url": "https://github.com/fastfetch-cli/fastfetch/releases/download/$version/fastfetch-$version-windows-i6i6.7z"
```
Comment (aliesbelik) [inline bucket/fastfetch.json:27]:
```suggestion
                "url": "https://github.com/fastfetch-cli/fastfetch/releases/download/$version/fastfetch-$version-windows-i686.7z"
```

**Review by aliesbelik — CHANGES_REQUESTED** [review]

**Comment by CarterLi** [comment]
oops

**Comment by CarterLi** [comment]
Any chance to merge it?

**Pull request merged**